### DISCUSSION TITLE
[Bug] Setting to hide Packages & Environments on Repo home page not saved

### DISCUSSION THREAD

**Discussion by Swiss-Mac-User** [community/community#62025] [Repositories] [answered]
### Select Topic Area

Bug

### Body

On the repository home page under Settings, there are checkboxes to hide „Releases“, „Packages“, and „Environments“.

However, I discovered that unchecking the later 2 of these is not saved and hence not applied. Reopening the Settings shows both as checked again.

* System: macOS 14 Sonoma
* Browser: Safari on macOS
* Browsing: non-private mode

**Comment by philipp-meier** [comment]
I think a similar bug was also reported here, but the cause of the problem seems to be different: https://github.com/orgs/community/discussions/62048

I do have the same problem with all my repositories lately. On repositories with the "Release area", I can't disable it and on others I can't enable it.
It seems like the request to `settings/update_meta` does only contain the fields "Description", "Website" and "Topics", but is missing the fields below "Include in the home page" in the payload.

**Edit (02.08.2023):** I just created a bug report for this at https://support.github.com/contact/bug-report.

**Comment by WhiteHoodHacker** [comment]
I can also confirm that the options under "Include in the home page" do not work. The request to `/repo_owner/repo_name/settings/update_meta` only includes the following form data fields:
```
_method: PUT
authenticity_token: ...
repo_description: ...
repo_homepage: ...
repo_topics[]: ...
```
The request does not include fields for any of the options below:
![image](https://github.com/community/community/assets/35712968/897acf76-fbfb-4cc3-a176-eda37954a637)

As a result, checking/unchecking any of those boxes doesn't do anything. I thought it might have been a bug with the new navigation feature preview, but it is broken on the old navigation as well.

**Reply by oliveratgithub** [reply]
Great analysis! Thanks a lot for adding this 👍

**Comment by nicfv** [comment]
Just made a post here about this: https://github.com/orgs/community/discussions/62115
Was accidentally a duplicate of this item. However, I did add a video screengrab in that issue in case that helps track down the bug.

**Comment by forrestwilkins** [comment]
This is also happening for my repo: https://github.com/praxis-app/praxis-api. It's not happening for any of my other repos though.

**Comment by haotian-liu** [comment]
Thanks for the analysis. This is happening to my repo as well -- I cannot turn on the release in my repo: https://github.com/haotian-liu/LLaVA

Hope this can be fixed soon :)

**Comment by hetabjorklund** [comment]
This is also happening to a new repo I created today (https://github.com/hetabjorklund/star-wars-app), I can't hide Releases, Packaged or Environments.

**Comment by muhd-umer** [comment]
The same is happening for my repo (https://github.com/muhd-umer/simcomm), however, editing the description, website, and tags, works as intended.

**Comment by cba85** [comment]
I also made a duplicate post about this bug: https://github.com/orgs/community/discussions/62378 

Releases, Packages and Environments blocs are always included in the home page, even if I uncheck the options to remove them.

It is happening on all of my repos in my case.

**Comment by FlorianLeChat** [comment, marked as answer]
![Screenshot_20230803_133027_Gmail.jpg](https://github.com/community/community/assets/26360935/8657d8a7-bdbc-42a6-972b-aa873a6789e5)

After getting a reply from support, the bug seems to have been fixed on my side.

**Comment by userari** [comment]
The issue still persist today 08/28/2023, I can hide anything that I want, but the problem is I can't see it on my own repository ( logged and logged out ) which works fines, but someone else can see them as long as they are logged in 🤡
